worker-languagedetection-container/testcases/automated/CAF_4002/dhivehi_oriya.yaml.testcase

```diff
@@ -68,35 +68,11 @@ expectedOutputData:
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "60"
+        - data: "100"
```

Comment:
Is Oriya no longer supported? If so remove this testcase and update any relevant documentation.